Sources/ArcGISToolkit/Components/Form/FormInputs/MultiLineTextEntry.swift

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import FormsPlugin
+import SwiftUI
+
+/// A view for text entry spanning multiple lines.
+struct MultiLineTextEntry: View {
+    /// The current text value.
+    @State private var text: String
+    
+    /// The form element that corresponds to this text field.
+    let element: FieldFeatureFormElement
+    
+    /// A `TextAreaFeatureFormInput` which acts as a configuration.
+    let input: TextAreaFeatureFormInput
+    
+    /// Creates a view for text entry spanning multiple lines.
+    /// - Parameters:
+    ///   - element: The form element that corresponds to this text field.
+    ///   - text: The current text value.
+    ///   - input: A `TextAreaFeatureFormInput` which acts as a configuration.
+    init(element: FieldFeatureFormElement, text: String?, input: TextAreaFeatureFormInput) {
+        self.element =  element
+        self.text = text ?? ""
```

Comment:
FYI: State shouldn't be taken as an initializer parameter and an initializer parameter should not be used to set state. From the documentation for [`State`](https://developer.apple.com/documentation/swiftui/state):

> Like you do for a [StateObject](https://developer.apple.com/documentation/swiftui/stateobject), declare [State](https://developer.apple.com/documentation/swiftui/state) as private to prevent setting it in a memberwise initializer, which can conflict with the storage management that SwiftUI provides. Unlike a state object, **always initialize state by providing a default value in the state’s declaration** [emphasis added], as in the above examples. Use state only for storage that’s local to a view and its subviews.

Reply:
This is set to be addressed in #365

Reply:
Understood. I read through some relevant threads ([1](https://forums.swift.org/t/why-swiftui-state-property-can-be-initialized-inside-init-this-other-way/62772/33), [2](https://forums.swift.org/t/default-init-internal-needs-a-workaround-enough-is-enough/40931/50)) on the Swift forums to try to deepen my understanding.

In the current state of the Form component it is expected that the `MultiLineTextEntry` and `SingleLineTextEntry` views will always be initialized with the same text values. Their parent view (`FormView`) is not notified of changes made by a user to the text value and we're not updating the feature's attributes.

> If the view is re-initialized with a different value for one of those parameters corresponding to a state property, the state property won't actually be updated to reflect that new value

This is coming in our next iteration of the Form component work. We'll be persisting any edits into the feature's attributes. At this point we'll likely revise how `MultiLineTextEntry` and `SingleLineTextEntry` are initialized because then `FormView` _will_ be concerned with edits made.